src/components/FeedbackCard.tsx

```diff
@@ -107,7 +111,7 @@ const FeedbackCard: React.FC<IProps> = ({
               </Button>
               <Button
                 variant="outline-color"
-                leftIcon={<ThumbsUp className="flip" />}
+                leftIcon={<ThumbsDown />}
```

Comment:
```suggestion
                leftIcon={<Icon as={ThumbsUp} transform="scaleY(-1)" fontSize="25px" />}
```

Reply:
@pettinarip going to take this over and get it over the line.